Libraries/Lists/FlatList.js

```diff
@@ -629,16 +655,54 @@ class FlatList<ItemT> extends React.PureComponent<Props<ItemT>, void> {
     }
   };
 
+  _listItemComponent = (info: RenderItemProps<ItemT>) => {
+    const {ListItemComponent, numColumns, columnWrapperStyle} = this.props;
+
+    if (!ListItemComponent) {
+      return null;
+    }
+
+    if (numColumns > 1) {
```

Comment:
Can you factor this shared code out into a separate function rather than copy-pasta?